### PR TITLE
Contribute/files

### PR DESCRIPTION
**Pull request by DashDeipayan**

### CODE REVIEW

**Comment by DashDeipayan** [comment]
@ankushdharkar please review my changes. I've updated the CONTRIBUTING.md file and README.md file to serve the current state purposes up to my knowledge of appropriation.

File: README.md [hunk 1]
```diff
@@ -19,8 +19,7 @@ members
 ### Rules for `rds-user-id` in Real Dev Squad:
 
 - Must begin with your first name
-- Must be unique
-- Must maintain alphabetic order in the `mappings` file
```
Comment (DashDeipayan) [inline README.md:23]: As there is no file by the name `mappings` I found this redundant.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 - Add your personal details in this repository so that it appears on the [members](https://members.realdevsquad.com/) page.
 
-# How do I my personal details?
+# Adding my profile picture for Members page
```
Comment (ankushdharkar) [inline CONTRIBUTING.md:5]: *your

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -48,37 +48,12 @@ git checkout -b develop
 
 5. **Just do it!**
 
-Add your RDS ID (you can choose any, if it doesn't exist before) and Github ID in `/ids/mapping.json` file **alphabetically**. Read rules for RDS ID [here](https://github.com/Real-Dev-Squad/website-static#rules-for-member_rds_id-in-real-dev-squad).
+Add your RDS ID (you can choose any, if it doesn't exist before).
```
Comment (ankushdharkar) [inline CONTRIBUTING.md:51]: We don't choose an ID here anymore. This sentence can be removed, right?

File: README.md [hunk 4]
```diff
@@ -19,8 +19,7 @@ members
...
+- Must be **unique**
```
Comment (ankushdharkar) [inline README.md:22]: This happens on the form now. Do we still need this here?

File: README.md [hunk 5]
```diff
@@ -29,6 +28,7 @@ members
 ### Rules for Profile picture:
 
 - Images should be a square
+- Image must be a `*.png`
```
Comment (ankushdharkar) [inline README.md:31]: `img.png` to be specific then

**Review by ankushdharkar — CHANGES_REQUESTED** [review]

**Comment by DashDeipayan** [comment]
I changed them accordingly. Kindly review the changes @ankushdharkar

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,8 +1,8 @@
-# How can I contribute?
+# How can you contribute?
 
-- Add your personal details in this repository so that it appears on the [members](https://members.realdevsquad.com/) page.
+- Add your personal details in this repository by signing in on our [website](https://www.realdevsquad.com) so that it appears on the [members](https://members.realdevsquad.com/) page.
```
Comment (swarajpure) [inline CONTRIBUTING.md:3]: Umm... When a user signs in on this website, only the user's `github_id` and `github_display_name` (if any) are stored in the database.
For users to be able to fill their personal details (first name, last name, username, company etc), we have got https://my.realdevsquad.com/signup
This form is the place where users can fill in their personal details which are required to be displayed on the members page

First the users will sign in via github on realdevsquad.com and then they should go to https://my.realdevsquad.com/signup to fill their profile

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -48,37 +48,10 @@ git checkout -b develop
 
 5. **Just do it!**
 
-Add your RDS ID (you can choose any, if it doesn't exist before) and Github ID in `/ids/mapping.json` file **alphabetically**. Read rules for RDS ID [here](https://github.com/Real-Dev-Squad/website-static#rules-for-member_rds_id-in-real-dev-squad).
-
-Make a new folder using your **RDS ID** in `members` folder.
+Make a new folder using your unique **RDS ID** in `members` folder.
```
Comment (swarajpure) [inline CONTRIBUTING.md:51]: Can we call this **RDS username** instead?

**Comment by DashDeipayan** [comment]
I have made the modifications according to the reviews. @swarajpure @ankushdharkar please review this PR.

**Review by swarajpure — APPROVED** [review]
Looks good to me.
Thanks!

**Pull request merged**